omniledger/service/service.go

```diff
@@ -707,15 +714,17 @@ func (s *Service) verifySkipBlock(newID []byte, newSB *skipchain.SkipBlock) bool
 	return true
 }
 
-// createStateChanges goes through all ClientTransactions and creates
-// the appropriate StateChanges. If any of the transactions are invalid,
-// it returns an error. If timeout is not 0, createStateChanges will stop
-// running instructions after that long, in order for the caller to determine
-// how many instructions fit in a block interval.
-func (s *Service) createStateChanges(coll *collection.Collection, scID skipchain.SkipBlockID, cts ClientTransactions, timeout time.Duration) (merkleRoot []byte, ctsOK ClientTransactions, ctsBad ClientTransactions, states StateChanges, err error) {
+// createStateChanges goes through all the proposed ClientTransactions and
```

Comment:
the comment says `ClientTransactions` but the function uses `TxResults`. which makes me wonder whether it's possible to have `createStateChanges` take `ClientTransactions`, because it doesn't need the boolean flag to do the job.